testing/prow/gpu-operator.sh

```diff
@@ -81,11 +81,21 @@ prepare_cluster_for_gpu_operator() {
 
     ${THIS_DIR}/entitle.sh
 
+    if [ "${USE_PREV_RELEASE_CATALOG:-}" ]; then
+        # Deploy OCP version N-1 catalog - Usually needed for testing on OCP next.
+        _warning "NFD_Operator_deployed_from_previous" "The NFD Operator was deployed from the version N-1 of the catalog (USE_PREV_RELEASE_CATALOG variable was set)"
```

Comment:
I think you don't need 2 warnings, `NFD_Operator_deployed_from_previous` and `GPU_Operator_deployed_from_previous`, that clutters the `ci-dashboard` dashboard, maybe have a single one `if [ "${USE_PREV_RELEASE_CATALOG:-}" ]; then _warning "operators_deployed_from_previous_catalog" "..."` ?

Reply:
I do want 2, We might have one operator from main catalog and another from a different one. Ill keep that for now.

Reply:
ack,
currently both of them are controlled `"${USE_PREV_RELEASE_CATALOG:-}"`, so it's either both or none, and new warnings can be added when the script is changed. But as you prefer of course

and I was thinking about adding a similar `_info` directory, that would be printed more lightly in `ci-dashboard`, but I never got to code it. Would be trivial in `ci-artifacts`, and fairly easy in `ci-dashboard`, if ever you want to give it a try ;-)